integration-tests/src/test/java/uk/gov/di/authentication/api/AuthenticationCallbackHandlerIntegrationTest.java

```diff
@@ -279,6 +288,77 @@ private void setupClientRegWithIdentityVerificationSupported() {
         }
     }
 
+    @Nested
+    class AccountInterventionJourney {
+
+        @BeforeEach()
+        void accountInterventionSetup() throws Json.JsonException {
+            setupTest();
+            setupSession();
+            setupClientRegWithoutIdentityVerificationSupported();
+        }
+
+        @Test
+        void shouldLogoutAndRedirectToBlockedPageWhenAccountIsBlocked() {
```

Comment:
Are we asserting anywhere that they're logged out?

Reply:
Have added validations that the session exists in redis before logout, and can't be retrieved afterwards. 

Additionally I noticed that we actually don't set up a client session attached to the session in this integration test. When I added the code to do this and attempted to validate that the client session is destroyed on logout, it appears to not get destroyed. Spent some time on it but it needs more investigation.

Reply:
To clarify on this, I was suggesting that:
- The validation to check the session exists in redis before logout, and can't be retrieved afterwards, is sufficient to resolve this conversation.
- The issue about not setting up a client session can be investigated in a separate ticket / PR.